src/transf.cpp

```diff
@@ -145,6 +145,7 @@ Constructs an uninitialized {} of degree ``0``.
           "copy",
           [](PyClass const& f) { return PyClass(f); },
           fmt::format(
+              // TODO the doc here isn't so good, there's actually no parameter
               R"pbdoc(
 Copy a {0}.
```

Comment:
Should this be `__copy__` to be used with `copy.copy`?

Reply:
Also, I think I was going for having both `copy` and `__copy__` but haven't done the second here yet, I think I'll do that in a separate PR, there are some more things that need to be fixed for transformations too.